crates/sargon/src/wrapped_radix_engine_toolkit/low_level/v2/transaction_manifest_v2/child_subintent.rs

```diff
@@ -0,0 +1,85 @@
+use crate::prelude::*;
+
+#[derive(Clone, Debug, PartialEq, Eq, uniffi::Record)]
+pub struct ChildSubintent {
+    pub hash: SubintentHash,
+}
+
+impl From<ChildSubintent> for ScryptoChildSubintent {
+    fn from(value: ChildSubintent) -> Self {
+        ScryptoChildSubintent {
+            hash: ScryptoSubintentHash {
+                0: value.hash.hash.into(),
+            },
+        }
+    }
+}
+
+impl From<(ScryptoChildSubintent, NetworkID)> for ChildSubintent {
+    fn from(value: (ScryptoChildSubintent, NetworkID)) -> Self {
+        Self {
+            hash: (value.0.hash, value.1).into(),
+        }
+    }
+}
+
+impl From<(ScryptoSubintentHash, NetworkID)> for SubintentHash {
+    fn from(value: (ScryptoSubintentHash, NetworkID)) -> Self {
+        Self::from_scrypto(value.0, value.1)
+    }
+}
+
+impl From<SubintentHash> for ScryptoSubintentHash {
+    fn from(value: SubintentHash) -> Self {
+        Self {
+            0: value.hash.into(),
```

Comment:
hmm never seen `{ 0: }` before, I think it is a new type? So try `Self(ScryptoHash::from(value.hash))` (also using the `TypeWeWant::from` instead of `into`...)